src/Filesystem/DTO/FileSize.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+namespace ILIAS\Filesystem\DTO;
+
+use ILIAS\Filesystem\Exception\IllegalArgumentException;
+
+/**
+ * Class FileSize
+ *
+ * This class provides the file size with additional information to
+ * remove the work to calculate the size to different unit like GiB, GB usw.
+ *
+ * @author  Nicolas Schäfli <ns@studer-raimann.ch>
+ * @since 5.3
+ * @version 1.0
+ */
+final class FileSize {
```

Comment:
Ok, then trust me and move it to the Data-dir =) This would be useful for disk quota, file upload sizes, inputs, ... Maybe it is DataSize, because it does not only talk about single files.